Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1750
+||kjos.vo.llnwd.net^
```

Comment:
Wasn't it enough to remove it just from Mobile Ads filter? - https://github.com/AdguardTeam/AdguardFilters/commit/9c8099d5f67bfb1231498a153d4d76cf52aa1432

Reply:
Let's check tomorrow. Same rule may be loaded from 3p sources.

Reply:
It seems removed from our DNS filter now.